src/System.Net.Http/src/System/Net/Http/CurlHandler/CurlHandler.SslProvider.Linux.cs

```diff
@@ -233,8 +233,12 @@ private static void SetSslVersion(EasyRequest easy, IntPtr sslCtx = default(IntP
                     case SslProtocols.Tls12:
                         curlSslVersion = Interop.Http.CurlSslVersion.CURL_SSLVERSION_TLSv1_2;
                         break;
+                    case SslProtocols.Tls13:
+                        curlSslVersion = Interop.Http.CurlSslVersion.CURL_SSLVERSION_TLSv1_3;
+                        break;
 
                     case SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12:
+                    case SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12 | SslProtocols.Tls13:
```

Comment:
Hmm.  Previously if you specified just Tls and Tls11, we wouldn't use TLSv1 because it also included Tls12.  But now if you specify Tls And Tls11 and Tls12, we'll still use TLSv1, but it might include Tls13 as well, which is an inconsistency.  I realize that's unrelated to your change, and I'm not sure what we can really do about it, as I assume libcurl doesn't provide an option for 1.0-1.2 and not 1.3.  So what you have is ok, it's just... odd.

Reply:
If you said 1.0 and 1.1 (but not 1.2) you'd get an error (the default case).  cURL is "one protocol, or you'll accept what I give you", so this is just saying that 1.0/1.1/1.2 is the old "everything/whatever", and 1.0-1.3 is the new one.

But, yeah, it's a weird layering